CRM/Upgrade/Incremental/sql/5.10.alpha1.mysql.tpl

```diff
@@ -1 +1,4 @@
 {* file to handle db changes in 5.10.alpha1 during upgrade *}
+
+{* Continuation from CRM-6405 it appears no upgrade step was done back in 3.2 *}
+UPDATE civicrm_mailing_bounce_type SET hold_threshold = 30 WHERE name = 'Away';
```

Comment:
You might consider a change from:

```sql
WHERE name = 'Away'
```

to

```sql
WHERE name = 'Away' AND hold_threshold = 3
-- Match on 3... or whatever the old default was...
```

Thoughts:

* v3.2.0 was released 8 years ago. That's a long time to miss out on a policy change. A lot of weird stuff can happen in 8 years. Like... people going in to edit the values of `hold_threshold` manually.
* Folks could reasonably disagree over whether `civicrm_mailing_bounce_type.hold_threshold` represents a contract or an obscure internal detail.
    * The existence/design of `civicrm_mailing_bounce_type.hold_threshold` strongly implies that different deployments can tweak the threshold -- thresholds which could become anything for any reason. (Maybe an older site bumped up to 20; maybe a younger site bumped up to 40 or bumped down to 15.) This upgrade step would (from their perspective) raise an [r-tech](https://docs.civicrm.org/dev/en/latest/standards/review/#r-tech) issue of trampling the customization.
    *  OTOH, there's no API or UI exposure of this thing, so I'd be surprised if 5 people on the planet had ever tweaked `hold_threshold`. It feels obscure.
    * We lack telemetry to demonstrate the significance or insignificance of existing data in `civicrm_mailing_bounce_type.hold_threshold`. It's ambiguous.
* Including the criterion `AND hold_threshold = 3` is the flawed-but-closest way I can see to saying "only update ancient, uncustomized records -- leave customized things alone". 

Personally, I'd use the tighter scoping (b/c I feel some sense of vigilance about upgrade-compatibility), but take this as a very soft recommendation - the matter is pretty ambiguous. You can continue or revise with whichever criterion you prefer. (I won't be offended by one or the other. ;) ) Either way, I'd allow brief pause (maybe a day or so) in case other/stronger comments come up.